third-party/realdds/include/realdds/topics/image/image-msg.h

```diff
@@ -15,23 +34,17 @@ class image
 public:
     using type = raw::device::imagePubSubType;
 
-    static std::string construct_topic_name( const std::string & topic_root, const std::string & stream )
```

Comment:
Why did you remove `construct_topic_name` here but left it on `notification-msg`?

Reply:
Because it needs refactoring, too, and I just didn't get to it yet. It'll get removed.